elastic/kibana
# [7.16] Ignore security disabled HTTP response header

## Summary

Close https://github.com/elastic/kibana/issues/115632

Do not show a warning toast that includes `Elasticsearch built-in security features are not enabled` substring. 
Note: needed only for 7.16 and 7.15

### How to test

You'll need a 7.16 deployment without explicitly disabled security. Go to Discover: in 7.16 you'll see warning toasts, in this pr we don't show them.


### Release note

Do not show security disabled warning when running searches
